tests/functional/object/mcg/test_store_status_transitions.py

```
@@ -0,0 +1,198 @@
+        )
+        uls_name = store.uls_name
+        logger.info(
+            f"Created {store_type} {store.name} backed by "
+            f"self-ref MCG bucket {uls_name}"
+        )
+
+        # 2. Verify the store is in Ready phase and OPTIMAL mode
+        assert_func(store.name, constants.STATUS_READY, BS_OPTIMAL, mcg_obj)
+
+        # 3. Delete the underlying MCG bucket
+        logger.info(f"Deleting underlying MCG bucket: {uls_name}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)